lib/email-addresses.js

```diff
@@ -528,7 +528,21 @@ function parse5322(opts) {
             obsGroupList
         )());
     }
-
+    
+    function fromSpec() {
+        return wrap('from', or(
+            mailboxList,
+            addressList
+        )());
+    }
+    
+    function senderSpec() {
+        return wrap('sender', or(
+            mailbox,
+            address
+        )());
+    }
+    
```

Comment:
I figured we don't need it because reply-to is just address-list, so it's superfluous.